install/installer/cluster_dev.py

```diff
@@ -372,6 +497,367 @@ def choose_git_provider(cli_arg, repo):
     return git_provider
 
 
+@typechecked
+def get_data_from_aws_config(login: str, password: str) -> {str, object}:
+    """
+    Get and parse config from file
+
+    Args:
+        login (str): CLI argument provided by user.
+        password (str): CLI argument provided by user.
+    Returns:
+        config ConfigParser obj|False: config, if exists. Otherwise - False
+    """
+    # Skip if CLI args provided
+    if login and password:
+        return False
+
+    try:
+        # Required mount: $HOME/.aws/:/home/cluster.dev/.aws/:ro
+
+        # User can have multiply creds, ask him what should be used
+        # https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#configuring-credentials
+        file = f'{os.environ["HOME"]}/.aws/credentials'
+        os.path.isfile(file)
+
+        config = ConfigParser()
+        config.read(file)
+
+    except FileNotFoundError:
+        return False
+
+    return config
+
+
+@typechecked
+def get_aws_config_section(config: {object, bool}) -> {str, bool}:
+    """
+    Ask user which section in config should be use for extracting credentials
+
+    Args:
+        config (ConfigParser obj|False): INI config
+    Returns:
+        config_section str|False: section name, if exists. Otherwise - False
+    """
+    # Skip if CLI args provided or configs doesn't exist
+    if not config:
+        return False
+
+    if len(config.sections()) == 0:
+        return False
+    elif len(config.sections()) == 1:
+        section = config.sections()[0]
+        print(f'Use AWS creds from file, section "{config.sections()[0]}"')
```

Comment:
```suggestion
        print(f'Use AWS creds from file, section "{section}"')
```

Reply:
You mean

```python3
section = config.sections()[0]
print(f'Use AWS creds from file, section "{section}"')
```
?

Reply:
Yep, that first line was already there when I first posted the suggestion :)